data/benefits/javascript/caf-eure-aide-bafa-generale.yml

```
@@ -17,13 +17,15 @@ conditions_generales:
   - type: quotient_familial
     floor: 600
     period: month
+  - type: regime_securite_sociale
+    includes:
+      - regime_general
 profils: []
-link: https://www.caf.fr/allocataires/caf-de-l-eure/offre-de-service/enfance-et-jeunesse/le-brevet-d-aptitude-aux-fonctions-d-animateur-bafa
-form: https://www.caf.fr/sites/default/files/caf/271/Documents/DEM_BAFA_BAFD.pdf
-instructions: https://www.caf.fr/sites/default/files/caf/271/Documents/Notice%20BAFA3.pdf
-prefix: l’
+interestFlag: _interetBafa
 type: float
 unit: €
 periodicite: ponctuelle
 montant: 250
-interestFlag: _interetBafa
+link: https://caf.fr/allocataires/caf-de-l-eure/offre-de-service/vie-professionnelle/le-brevet-d-aptitude-aux-fonctions-d-animateur-bafa-et-aux-fonctions-de-directeur-bafd
```

Le lien n'est plus valide. J'ai trouvé ça: 
```suggestion
link: https://caf.fr/allocataires/caf-de-l-eure/offre-de-service/vie-professionnelle/la-caf-vous-aide-financer-le-bafa
```

---

C'est modifié @Valandr oui, le lien que tu as mis est le bon. :)